backend/nexus-typegen.ts

```diff
@@ -0,0 +1,200 @@
+import { LocationModel } from "./src/dataAccess/dataTypes" 
+
+import type { Context } from "./src/api/context"
+import type { core } from "nexus"
+declare global {
+  interface NexusGenCustomInputMethods<TypeName extends string> {
+    /**
+     * A date-time string at UTC, such as 2007-12-03T10:15:30Z, compliant with the `date-time` format outlined in section 5.6 of the RFC 3339 profile of the ISO 8601 standard for representation of dates and times using the Gregorian calendar.
+     */
+    datetime<FieldName extends string>(fieldName: FieldName, opts?: core.CommonInputFieldConfig<TypeName, FieldName>): void // "DateTime";
+  }
+}
+declare global {
+  interface NexusGenCustomOutputMethods<TypeName extends string> {
+    /**
+     * A date-time string at UTC, such as 2007-12-03T10:15:30Z, compliant with the `date-time` format outlined in section 5.6 of the RFC 3339 profile of the ISO 8601 standard for representation of dates and times using the Gregorian calendar.
+     */
+    datetime<FieldName extends string>(fieldName: FieldName, ...opts: core.ScalarOutSpread<TypeName, FieldName>): void // "DateTime";
+  }
+}
+
+
+declare global {
+  interface NexusGen extends NexusGenTypes {}
+}
+
+export interface NexusGenInputs {
+}
+
+export interface NexusGenEnums {
+  LocationEnum: 999 | 15 | 8 | 2 | 18 | 3 | 14 | 22 | 23 | 19 | 10 | 4 | 9 | 7 | 12 | 1 | 13 | 11 | 16 | 21 | 6 | 17 | 20 | 0 | 5 | 24
```

Comment:
Why aren't these ordered numerically?

Reply:
This is generated by Nexus [based on this declaration](https://github.com/Makermed/CleanAirMap/blob/backend-draft/backend/src/api/graphql/Location.ts#L5).

It looks to me as if they are ordered alphabetically by the associated String values (ALL, ARENA, ...), which are generated in the GraphQL schema [here](https://github.com/Makermed/CleanAirMap/blob/fb046c9049cc3348775be7b58c4531d800094035/backend/schema.graphql#L29).

I'm honestly not sure where the translation magic happens. Somewhere in Apollo I think, because the input string from the mutation is already translated to the associated numeric value by the time it reaches [the resolver](https://github.com/Makermed/CleanAirMap/blob/fb046c9049cc3348775be7b58c4531d800094035/backend/src/api/graphql/Location.ts#L101)